modules/mesos-cluster/main.tf

```diff
@@ -229,3 +229,15 @@ resource "aws_route53_record" "paas-mozilla-org-dns-zone" {
   ttl = 300
   records = ["${aws_elb.mozilla-org-elb.dns_name}"]
 }
+
+resource "aws_cloudwatch_metric_alarm" "cloudwatch-health-host-count-alarm" {
```

Comment:
We also need to tell this alarm what actions to take if it goes off. We can do this by speficying the [ARN](http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) for our SNS topic in `alarm_actions`